src/components/Book.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { deleteBook, fetchBooks } from '../redux/Books/booksSlice';
+import styles from '../styles/Book.module.css';
+
+const Book = ({ booklListObject }) => {
+  const dispatch = useDispatch();
+
+  const handleDeleteBook = async (id) => {
+    try {
+      await dispatch(deleteBook(id));
+      dispatch(fetchBooks());
+    } catch (error) {
+      // Handle error if needed
+    }
+  };
+
+  return (
+    <div className={styles.bookcardcontainer}>
+      <div className={styles.bookinfocontainer}>
+        <div className={styles.maininfocontainer}>
+          <div className="Book_maininfocontainer__JPhJd">
+            <p className="Book_categorycontainer__YmVid">{booklListObject.category}</p>
+            <h2 className="Book_titlecontainer">{booklListObject.title}</h2>
+            <h6 className="Book_authorcontainer">{booklListObject.author}</h6>
+          </div>
+          <ul className={styles.buttonscontainer}>
+            <li>
+              <button type="button" className={styles.button1}>
+                Comment
+              </button>
+            </li>
+            <li>
+              <button
+                type="button"
```

Comment:
- [ ]  As requested from the requirement, you design should exatctly match with the one provided from the design and the colors used as well. Please check this part and fix to match with the one from the design:
Your design 
![image](https://github.com/kayjeee/ceactbookstore/assets/43877689/7fb0cf8b-b682-4d75-a1c3-032eca641675)

Provided reference design 
![image](https://github.com/kayjeee/ceactbookstore/assets/43877689/2b094246-b51e-4ca3-a967-19c46a29b24f)

- [ ]  I think that your buttons are also a bit different from the design as well, especial the border radius and the size, make sure that it 100% match with the buttons from the design

Reply:
hi, @ishimwezachee 
let us follow each other on Git Hub.
My GitHub profile: [Kayjee](https://github.com/kayjeee)
<img src="https://media3.giphy.com/media/21I909B1Us5hsGiupn/giphy.gif"/>